src/tests/base_test.py

```diff
@@ -78,6 +82,41 @@ def update_parameterized(cls, request: pytest.FixtureRequest, config: BaseConfig
                 else:
                     raise AttributeError(f"No attribute name {fixture_name} in {config_type} object type")
 
+    @pytest.fixture
+    def k8s_minikube(self, request: pytest.FixtureRequest) -> kubernetes.dynamic.DynamicClient:
+        """k8 minikube wrapper client  object fixture passed to test function
+        This function fixture called by testcase, we initialize k8client
+        Storing configmap before test begins -> for later restoring
+        When test func done, check if configmap was changed and restore config.
+        :return:
+        """
+        log.debug(f"--- SETUP --- Creating k8s_minikube for test: {request.node.name}\n")
+        k8_client_ = kubernetes.dynamic.DynamicClient(
+            kubernetes.client.api_client.ApiClient(
+                configuration=kubernetes.config.load_kube_config(),
```

Comment:
Sounds good to me , i see that we have a factory class to initilize client, i did not dig into it yet because i dont use it but i guess that the way we can do it is another fixture called from minikube ... 

Not changing it now , any way its may be extended IMO.

@pytest.fixture
    def k8s_minikube(self, k8s_client request: pytest.FixtureRequest)